storage/connectors/cassandra/src/main/java/feast/storage/connectors/cassandra/retriever/CassandraOnlineRetriever.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright 2018-2021 The Feast Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package feast.storage.connectors.cassandra.retriever;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.cql.Row;
+import com.datastax.oss.driver.api.core.cql.SimpleStatement;
+import com.datastax.oss.driver.api.querybuilder.QueryBuilder;
+import com.datastax.oss.driver.api.querybuilder.select.Select;
+import com.datastax.oss.driver.api.querybuilder.select.Selector;
+import com.google.protobuf.ByteString;
+import feast.proto.serving.ServingAPIProto;
+import feast.proto.types.ValueProto;
+import feast.storage.api.retriever.Feature;
+import feast.storage.api.retriever.OnlineRetrieverV2;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class CassandraOnlineRetriever implements OnlineRetrieverV2 {
+
+  private CqlSession session;
+  private CassandraSchemaRegistry schemaRegistry;
+
+  private static String ENTITY_KEY = "key";
+
+  public CassandraOnlineRetriever(CqlSession session) {
+    this.session = session;
+    this.schemaRegistry = new CassandraSchemaRegistry(session);
+  }
+
+  /**
+   * Generate name of Cassandra table in the form of <feastProject>__<entityNames>
+   *
+   * @param project Name of Feast project
+   * @param entityNames List of entities used in retrieval call
+   * @return Name of Cassandra table
+   */
+  private String getTableName(String project, List<String> entityNames) {
+    String tableName =
+        String.format("%s__%s", project, entityNames.stream().collect(Collectors.joining("__")));
```

Comment:
Table name is different from bigtable, i think i used single underscore instead of two

Reply:
Should we standardize both to use 2 underscores instead?